src/abilities/Dark-Priest.js

```diff
@@ -274,7 +274,7 @@ export default (G) => {
 					return item.isWalkable(crea.size, 0, false);
 				});
 
-				spawnRange = arrayUtils.extendToLeft(spawnRange, crea.size);
+				spawnRange = arrayUtils.extendToLeft(spawnRange, crea.size, G.grid);
```

Comment:
In other commits it looks like you finished removing the global `G` var?

Reply:
It is actually removed. Abilties now return a function that requires the game (As a G argument for minimal line changes to minimize merge conflicts)